website/events/2021-08-19-weekly-report-2021-8.9-8.15.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Apache APISIX 社区周报 ｜ 2021 8.9-8.15
+---
+
+<!--truncate-->
+
+## **导语**
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+
+我们还整理了一些适合新来社区的小伙伴们参加的 issue ！感兴趣的同学们，走过路过不要错过！
+
+## 贡献者统计
+
+本周，有 26 位开发者为 Apache APISIX 提交了  96  个 commit 。感谢本周以下小伙伴为 Apache APISIX 添砖加瓦（排名不分先后），是你们的无私付出，让 Apache APISIX 变得更好！
+> juzhiyuan, Baoyuantop, spacewander, okaybase, liuxiran, Serendipity96, nic-chen, tao12345666333, shuaijinchao, Yiyiyimu, jbampton, tzssangglass, foolwc, dickens7, Xunzhuo, jagerzhang, tong3jie, sunhao-java, gxthrj, chzhuo, qsliu2017, bisakhmondal, LiteSun, wmdmomo, Ylikj, bzp2010
+## Good first issue
+
+#### Issue #4800

```

Comment:
it would be better to use list for every good first issue, together with its link

- [Issue #4800](link)